templates/plugin/salt_master.conf.erb

```diff
@@ -0,0 +1,58 @@
+# /etc/salt/master.d/foreman.config Example configuration
+#
+# This file summarizes configurations for the salt-master. Modify directories and
+# parameters to fit your setup. When you're done, remove the .example from the
+# filename so the salt-master will make use of it.
```

Comment:
This feels like bad instructions for the user.

Reply:
Shortened to a single sentence. Any comments/suggestions?